src/funcs/transformTableStructureToDBML.js

```diff
@@ -24,21 +24,17 @@ const getColumnType = (col) => {
   return columnType;
 }
 
-const cleanUpColumnDefault = columnDefault => columnDefault && columnDefault.includes('::text')
-  ? columnDefault.replace(/::text/gi, '').replace(/'/gi, '')
-  : columnDefault;
 
 const getColumnDefault = (columnDefault, dataType) => {
-  if (!columnDefault) return '';
-  const cleanedUp = cleanUpColumnDefault(columnDefault);
+  if (!columnDefault || columnDefault.includes('::')) return '';
```

Comment:
So this line here actually fixes the bug described by HPAPI-308, correct?

Reply:
well yes and no. Yes, as in, you can now just copy n paste the DBML to db diagram and it should not have any issues (at least with our db, cannot speak broadly 😄 ). The DBML issue was having an extra set of quotes inside a string. This made it seem like some of the strings were keywords DBML should know about. This happened on strings being returned from Postgres that showed what was going on under the hood. All of those strings had the `::` in them. so to kill 2 birds with 1 stone, I decided I saw no need to show end users of the db tool what Postgres does to auto-increment an id.